### PR TITLE
Project Portfolio

### PR DESCRIPTION
**Pull request by TanYikai**

### CODE REVIEW

File: docs/team/projectPortfolio.adoc [hunk 1]
```diff
@@ -0,0 +1,119 @@
+= Tan Yikai - Project Portfolio
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+== Project: AddressBook - Level 4
```
Comment (joanneong) [inline docs/team/projectPortfolio.adoc]: Wrong project, go and copy Zhen Yong's

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -72,7 +73,7 @@ Examples:
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 * `add n/Betsy Crowe a/Newgate Prison p/1234567 e/betsycrowe@example.com`
 * `add n/Betsy Crowe`
-
+// end::EnhancedAdd[]
 === Adding a relationship between two persons: `addRelationship`
```
Comment (wenmogu) [inline docs/UserGuide.adoc]: please put a blank line there

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -97,11 +98,12 @@ Examples:
...
 Format: `list`
 
+// tag::sort[]
 === Sorting all persons : `sort`
 
 Shows a list of all persons in the address book sorted alphanumerically by name. +
 Format: `sort`
-
+// end::sort[]
 === Editing a person : `edit`
```
Comment (wenmogu) [inline docs/UserGuide.adoc]: same here

File: docs/team/projectPortfolio.adoc (hunk 1 above)
Comment (wenmogu) [inline docs/team/projectPortfolio.adoc]: it shld be intelliJ...this description can copy from others' portfolio in the main repo

File: docs/team/projectPortfolio.adoc [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+AddressBook - Level 4 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 6 kLoC.
+
+*Code contributed*: [https://github.com/CS2103AUG2017-F10-B1/main/tree/master/collated[Functional code]] [https://github.com/CS2103AUG2017-F10-B1/main/tree/master/collated[Test code]] {give links to collated code files}
```
Comment (wenmogu) [inline docs/team/projectPortfolio.adoc]: the code link to be the link to ur collated code file, so it shld be something like https://github.com/CS2103AUG2017-F10-B1/main/blob/master/collated/main/TanYikai.md

File: docs/team/projectPortfolio.adoc [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+It is useful when the addressbook has alot of entries and allows user the option to organize their addressbook for a more pleasant appearance of the addressbook data. +
+
+It is most useful in providing user with the overall content of the addressbook in an organised manner which groups those with similar names together. +
+
+==== Implementation
+
+---
+#Start of Extract [from: Developer Guide]#
+
+include::../DeveloperGuide.adoc[tag=]
```
Comment (wenmogu) [inline docs/team/projectPortfolio.adoc]: what happened to the tag...

**Review by wenmogu — CHANGES_REQUESTED** [review]

**Review by wenmogu — APPROVED** [review]

**Pull request merged**